server/httpHandler.go

```diff
@@ -37,6 +40,26 @@ var postActionIntegrationRequestFromJson = model.PostActionIntegrationRequestFro
 var notifyHandlerError = defaultNotifyHandlerError
 
 func (p *Plugin) handleHTTPRequest(w http.ResponseWriter, r *http.Request) {
+	if r.URL.Path == "/giphyAttribution" {
```

Comment:
By putting the attribution file in a `/public` directory (created at root level), it should automatically be made available as a static resource, without having to handle it manually. The corresponding URL would be `fmt.Sprintf("/plugins/%s/public/%s", manifest.Id, <filename>)`.

Reply:
Wow, much cleaner implementation!